#This message is used to hold the positional error of the turtle being controlled.

float64 x_error #absolute value of the x error
float64 y_error #absolute value of the y error
float64 theta_error #absolute value of the angular error
